md/release-notes.md

```diff
@@ -415,16 +432,24 @@ Please use the Tomcat bundle instead, or see the [Custom Deployment](deploy-bund
 * BST-126 Studio must be restarted to use User-defined functions
 * BST-132 Wrong script generated in some specific cases with multiple data
 * BST-162 Generate process documentation does not work post 7.7
+* BST-201 Contract input name should never be equals to the associated business data name
+* BST-202 Application descriptor Deploy button does not trigger the saved changes and doesn't raise errors
 * BST-204 Default Application created in the Portal raises an error about custompage_home when imported and Deployed from Studio
 * BST-221 UI Designer allows to have 2 forms with same name when created from the Studio: this breaks LA Builder
+* BST-251 manage organizations wizard display issue
```

Comment:
not precise enough. What was the issue?

Reply:
A table  sometime very small...

Reply:
But it has been fixed like 4 months ago, what does this have to do in the 7.9.2 release note?

Reply:
If you look carefully, it's in the 7.9.0 section: at the time of 7.9.0, many bug fixes had no fixversion.
They have been added recently, so I'm catching up on the release notes from 7.9.0.
Not mentionning the 7.8.x bug fixes that have a pretty vague fixversion (like 7.8.x....).
We need to do better on fix versions to avoid such issues.

Reply:
changed in Jira and release notes to "On MacOS, manage organizations wizard needs scrolling to add a membership"